### PR TITLE
push notification subscription added

### PR DESCRIPTION
**Pull request by eugepae**

### CODE REVIEW

File: src/Softeq.NetKit.Chat.Web/Controllers/MemberController.cs [hunk 1]
```diff
@@ -120,5 +121,46 @@ public async Task<IActionResult> UpdateUserNotificationSettingsAsync(Notificatio
         }
 
         #endregion
+
+        #region Notifications
+        
+        [ProducesResponseType(typeof(void), StatusCodes.Status200OK)]
+        [HttpPost]
+        [Route("push-token/subscribe")]
+        public async Task<IActionResult> SubscribePushToken([FromBody]PushTokenRequest model)
```
Comment (DmitryMorozov228) [inline src/Softeq.NetKit.Chat.Web/Controllers/MemberController.cs]: Let's rename to SubscribePushTokenAsync

File: src/Softeq.NetKit.Chat.Web/Controllers/MemberController.cs [hunk 2]
```diff
@@ -120,5 +121,46 @@ public async Task<IActionResult> UpdateUserNotificationSettingsAsync(Notificatio
...
+            });
+
+            return Ok();
+        }
+
+        [ProducesResponseType(typeof(void), StatusCodes.Status200OK)]
+        [Route("push-token/unsubscribe")]
+        [HttpPost]
+        public async Task<IActionResult> UnsubscribePushToken([FromBody]PushTokenRequest model)
+        {
```
Comment (DmitryMorozov228) [inline src/Softeq.NetKit.Chat.Web/Controllers/MemberController.cs]: Let's rename to UnsubscribePushTokenAsync

File: src/Softeq.NetKit.Chat.Web/Controllers/MemberController.cs [hunk 3]
```diff
@@ -120,5 +121,46 @@ public async Task<IActionResult> UpdateUserNotificationSettingsAsync(Notificatio
...
+        #region Notifications
+        
+        [ProducesResponseType(typeof(void), StatusCodes.Status200OK)]
+        [HttpPost]
+        [Route("push-token/subscribe")]
+        public async Task<IActionResult> SubscribePushToken([FromBody]PushTokenRequest model)
+        {
+            var userId = GetCurrentSaasUserId();
+
+            var token = model.Token.Trim().Replace(" ", string.Empty);
```
Comment (DmitryMorozov228) [inline src/Softeq.NetKit.Chat.Web/Controllers/MemberController.cs]: Why we use Trim() here?

Comment (eugepae) [reply src/Softeq.NetKit.Chat.Web/Controllers/MemberController.cs]: because we are faced with spaces in device token received from mobile

**Pull request merged**